production/helm/loki/values.yaml

```diff
@@ -1124,10 +1124,12 @@ gateway:
       {{ if .Values.loki.tenants }}
 
 
+
```

Comment:
@vlad-diachenko i think this is a bug in your update 😊 it works really well but the new lines seems wrong.

Reply:
It seems to add a new line each time. 

How shall we proceed, @vlad-diachenko